### ISSUE TITLE
Colors of find results in comments make text impossible to read

### ISSUE THREAD

**Issue by Gitoffthelawn** [stylish-userstyles/stylish#295]
Issue: The colors used by the Find operation make it impossible to read some of the results

To reproduce this issue:
1. Create a new style
2. Enter text `/* a rose is a rose is a rose */`
3. Search for the string `rose`
4. The text of the second and third roses is illegible because the foreground color is almost the same as the background color

**Comment by un1versal** [comment]
yep syntax highlight in this really not great. I can indeed confirm this.

edit

this is what I see in mine, really not ideal legibility.
The gif below shows the 3 types of highlighting that imo illustrate the issue well.

1) find (focus)
2) find (not focus)
3) selection

![findselection](https://cloud.githubusercontent.com/assets/3521959/18660499/ae6e4aec-7f08-11e6-80e7-da409fa3c909.gif)

**Comment by un1versal** [comment]
I was digging around some-more with Firefox themes, and Im unsure if the syntax highlighter is part of Stylish or Firefox itself.

What I noticed is with a Firefox light editor theme, this isnt so much of a problem, but of course then you must wear sunglasses.

**Comment by un1versal** [comment]
more bad news, with latest Firefox 49, the dark style changed and now it even more unreadable for these parts, imo its now pretty much unusable as a dark style.

Im going to see if thers an easier way to override that bs localy.

**Comment by Gitoffthelawn** [comment]
@un1versal Yes, the Mozilla developers really screwed up the dark theme in Firefox 49.  They made it low contrast, which is _very_ hard to read.

They clearly have no vision experts on their team, or are ignoring them.  This is why companies need to hire UI/UX experts and actually listen to them.

**Comment by un1versal** [comment]
@Gitoffthelawn thats not quite the issue...

in any case here is some workaround for this https://userstyles.org/styles/107374/stylish-2-0-6-dark-theme

it needs tweaking, but it should solve this issue and perhaps there are other styles as well that suit your taste better.

~~the scrollbars in that even though they are fuggly as hell also fix the scrollbar bug in https://github.com/JasonBarnabe/stylish/issues/296~~

**Comment by un1versal** [comment]
Im in the process of making my own theme for this since the existing all suck.

**Comment by Gitoffthelawn** [comment]
@un1versal Thanks.  Let us know where you publish your theme!

**Comment by un1versal** [comment]
I wont publish it in stylish website, if anything will be a gist.

**Comment by Gitoffthelawn** [comment]
@un1versal Yes, a gist is fine.  Why not on stylish website out of curiosity (I assume you mean userstyles.org)?

**Comment by un1versal** [comment]
because I simply dont want to be responsible for maintenance and support. Already have too many projects.

**Comment by hideheader** [comment]
Stylish simply wraps the Firefox code editor. If you've a problem with the CodeMirror component per se, then report it to Mozilla.
https://bugzilla.mozilla.org/

**Comment by un1versal** [comment]
> Stylish simply wraps the Firefox code editor. If you've a problem with the CodeMirror component per se, then report it to Mozilla.
> https://bugzilla.mozilla.org/

Ide like to agree with you, but considering your comment, it admits there is a fault with Stylish and the way it does this _wrapping_ considering how unusable it is.

Ive solved the problem locally for me, and I dont feel like chasing mozilla for a game of pass the bug around. If anyone likes to report it feel free.

@Gitoffthelawn here I assume you know how to use this in Stylish? https://gist.github.com/un1versal/c76a69f44b84da7df885a7a696309f4c

If you dont like it, fix it, if it doesnt work or it eats your babies and kills your kittens, I also wave responsibility, if it causes thermonuclear war and the world ends, you can blame "The Flying Spaghetti Monster", You have been warned!

**Comment by Gitoffthelawn** [comment]
@un1versal Thanks.  The kittens already ate the babies, resulting in thermonuclear war during which the Flying Spaghetti Monster did not survive.

**Comment by un1versal** [comment]
> ...during which the Flying Spaghetti Monster did not survive.

impossible! ;) pics or it didnt happen ;D

**Comment by Gitoffthelawn** [comment]
@un1versal It definitely happened: http://imgur.com/nflEUBF  :smile:

**Comment by hideheader** [comment]
>Ide like to agree with you, but considering your comment, it admits there is a fault with Stylish and the way it does this wrapping considering how unusable it is.

Your stupidity offends me. Stylish merely makes use of the same editor component used everywhere in Firefox. If you've a problem with that component, report the problem to Mozilla. Whining about it here won't change it.
